src/app/profile/update/update.component.html

```diff
@@ -87,23 +87,50 @@ <h1>Edit Profile</h1>
     <!-- right column -->
     <div class="col-xs-12 col-sm-4 col-sm-offset-1">
       <form #advancedForm="ngForm">
-        <div class="form-group padding-top-40">
-          <div class="feature-toggle-text">
-            <div class="feature-toggle-text-header">
-              <h2>Experimental Features</h2>
-              <p>Experimental features are areas of the product that we’re trying out and would like your feedback on.</p>
-            </div>
-            <div class="feature-toggle-text-body">
-              <p>Experimental features are provided for use with no guarantee of service, data longevity, or support.  You agree to use these at your own risk with the acknowledgement that the features may not operate correctly and may be substantially modified.  These features are provided “AS IS” without warranty of any kind.  In no event shall Red Hat be liable for any damage whatsoever arising out of the use or inability to use these features. You can opt out of these experimental features at any point in time by unchecking the box below.</p>
-              <p>These terms and conditions do not replace the terms and conditions you agreed to when you signed up for the OpenShift.io service.</p>
```

Comment:
Is this legal text that still needs to be in the page somewhere?

Reply:
@corinnekrych I completely missed this. @dlabrecq is correct - this needs to be the listed text for the Experimental Features.

Reply:
@dlabrecq @mindreeper2420 good catch, I've just added it following invision screens:
https://redhat.invisionapp.com/share/4EEE2JAQ7#/screens/270538837